js/app.js

```diff
@@ -0,0 +1,33 @@
+$(document).ready(function() {
+
+
+   $('#submit-btn').click(function(event){ 
+      event.preventDefault();
+      var city = $('#city-type').val();
```

Comment:
Not necessarily "wrong" but you could have added this `$('#city-type').val('');` to make sure the field empties itself after each query.